src/keychain/index.js

```diff
@@ -503,6 +503,57 @@ class Keychain {
       return throwDelayed(errcode(new Error(`Key '${name}' does not exist. ${err.message}`), 'ERR_KEY_NOT_FOUND'))
     }
   }
+  
+  /**
+   * Rotate keychain password and re-encrypt all assosciated keys
+   *
+   * @param {string} oldPass - The old local keychain password
+   * @param {string} newPass - The new local keychain password
+   */
+  async rotateKeychainPass (oldPass, newPass){
+    if (typeof oldPass !== 'string' || typeof newPass !== 'string') {
+      throw new Error(`Invalid pass type '${typeof oldPass}'`);
```

Comment:
Probably not essential, but what do you think about using `throwDelayed` like the rest of the keychain functions? And please add an `errCode`